### DISCUSSION TITLE
Can you stop the "Coming Soon 👀" posts?

### DISCUSSION THREAD

**Discussion by sovetski** [community/community#61061] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi,

What are these kind of posts posted by the github team for?

https://github.com/orgs/community/discussions/60791
https://github.com/orgs/community/discussions/60802

It's just a few examples, I've already seen several times, especially posted by @tuves 

If you have news to announce, please add at least small information instead of posts with a single polluting title.

@tuves , you had already posted the exact same type of post at the end of May and the beginning of June several times, and then you deleted them. I imagine it's because you couldn't meet the deadlines, but in this case please don't spoil anything, even if apart from the title the posts mean nothing.

As loyal users some of us expect something when you share this kind of posts, so PLEASE STOP IT OR GIVE MORE INFORMATIONS

Thanks!!

**Comment by tuves** [comment, marked as answer]
Hi @sovetski,

Thank you for your feedback. The reason for these temporary posts is because we need a live link to update changelogs/blogs. Unfortunately, there is currently no way for us to hide posts or save them as drafts until the ship date. I have not deleted them in May or June, I've simply updated them with relevant product release info once we've gone live.

The reason for "Coming Soon" is to avoid spoiling anything, which is also why we don't add labels to give anything away. To respect the Product teams and their releases, we cannot give more information than this. I try to put them out as close to the ship date as possible, but unfortunately, it means there will be a few days in between when the post is in limbo. I understand this might be frustrating or build curiosity, however, it's the only manual stopgap that we currently have.

**Reply by spenserblack** [reply]
:thinking: Perhaps a different title can be used? The first time I saw one of these discussions, I thought to myself "oh, I should watch this, something exciting is coming!" But then [that discussion was closed and locked with no further updates :upside_down_face:](https://github.com/orgs/community/discussions/60802).
The :eyes: emoji really signalled to me that I should watch the discussion, so I was a bit confused about when it was closed with no further updates, until I stumbled on this discussion.

Maybe a title like ":memo: \<CHANGELOG PLACEHOLDER\>" would be less confusing. And/or the body could be something like "we will place our scheduled update here at the end of the month(?)" instead of just "Stay tuned!" A bit less fun :laughing:, but a bit more informative.

**Reply by sovetski** [reply]
Thank you @tuves for your answer it is more clear now!

I totally agree @spenserblack suggestion, other people like us, who will not see this subject in the future, will surely be wrong too

**Reply by tuves** [reply]
Noted @spenserblack, we can amend the title to these placeholders. 

Re: the post you linked, another teammate of mine is also prepping on an upcoming release, and the main reason we lock them before the announcement is to mitigate the spam. Since there is nothing notable to the post yet, it doesn't really warrant a text response.

Re: the body of the post saying something like "we will place our scheduled update here at the end of the month", in an ideal world, we wanted to say something similar, as we do have tentative dates for ships--however, it's not uncommon to run into delays, freezes, etc. So as a middle ground, we can put something like "stay tuned for more details to follow". 

Again, thanks for your feedback--I've shared this with the team.

**Comment by xieyuheng** [comment]
Please bring back the old feed!

**Comment by xieyuheng** [comment]
We need old feed back:

https://github.com/orgs/community/discussions/66244

**Comment by xieyuheng** [comment]
We need old feed back:

https://github.com/orgs/community/discussions/66244

**Comment by sovetski** [comment]
> Please bring back the old feed!

you are on the wrong discussion

**Reply by xieyuheng** [reply]
Where is the right discussion？
